.gitattributes

```diff
@@ -0,0 +1,7 @@
+# See https://git-scm.com/docs/gitattributes for more about git attribute files.
```

Comment:
- [ ] It would be better if we make our README file descriptive of the project. Currently, it has the default template from Rails. However, we can fix this by using the [recommended template](https://github.com/microverseinc/readme-template/blob/master/README.md), let us also keep in mind to make it descriptive although we still have the template

Reply:
Hi, @addod19! Thanks a lot for noting this! The README is now updated.😁